packages/core/test/unit/configuration.test.ts

```diff
@@ -193,11 +193,11 @@ describe('Sequelize constructor', () => {
     });
 
     it('priorises the ?host querystring parameter over the rest of the URI', () => {
-      const sequelize = new Sequelize(`${dialect}://localhost:9821/dbname?host=example.com`);
+      const sequelize = new Sequelize(`${dialect}://localhost:9821/dbname?host=/tmp/mysocket`);
```

Comment:
can you add a test that uses the socket path as an encoded domain?

Reply:
Like `${dialect}://encodeURIComponent(/tmp/mysocket):9821/dbname`?

Reply:
Yep!

Reply:
Done